Preparation/MyStack/MyStack.csproj

```diff
@@ -1,8 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You don't need changes in next files:
- Preparation/MyStack/MyStack.csproj
- Preparation/Preparation/Spiral_Array.csproj
- Preparation/Sand_Clock/Sand_Clock.csproj